ui/plugins/annotations/controller.js

```diff
@@ -42,7 +38,7 @@ treeherder.controller('AnnotationsPluginCtrl', [
             var jobMap = ThResultSetStore.getJobMap($rootScope.repoName);
             var job = jobMap[key].job_obj;
 
-            job.failure_classification_id = 1;
+            $scope.$evalAsync(function() {job.failure_classification_id = 1;});
```

Comment:
ensures the job up in the normal jobs view gets updates as to classified or not after this change.  The re-selecting handled that before.